### ISSUE TITLE
Not able to depend on app.cash.paging:paging-runtime-composeui

### ISSUE THREAD

**Issue by Shabinder** [cashapp/multiplatform-paging#97]
I am implementing the paging in a new project android and jvm, which didnt had androidx/multiplatform paging before.

`implementation("app.cash.paging:paging-runtime-composeui:3.2.0-alpha04-0.2.0")`

when I have above in common.main,

:android:checkDebugDuplicateClasses always **fails**, with following:

```
Execution failed for task ':android:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
Duplicate class androidx.paging.compose.LazyPagingItems found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItems$collectLoadState$2 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItems$collectPagingData$2 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItems$differCallback$1 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItems$pagingDataDiffer$1 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItemsKt found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItemsKt$collectAsLazyPagingItems$1 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItemsKt$collectAsLazyPagingItems$1$1 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItemsKt$collectAsLazyPagingItems$2 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
Duplicate class androidx.paging.compose.LazyPagingItemsKt$collectAsLazyPagingItems$2$1 found in modules paging-compose-1.0.0-alpha18-runtime (androidx.paging:paging-compose:1.0.0-alpha18) and paging-compose-common-jvm-3.2.0-alpha04-0.2.0 (app.cash.paging:paging-compose-common-jvm:3.2.0-alpha04-0.2.0)
```

I can confirm if I dont have above dependency, then there is no androidx.paging:paging-compose:1.0.0-alpha18 in any of my configurations.

to confirm that I have check deps tree and also tried with :
```
implementation("app.cash.paging:paging-runtime-composeui:3.2.0-alpha04-0.2.0") {
    exclude("androidx.paging","paging-compose")
}
```
which will comile fine but will throw when run:
``` 
java.lang.NoSuchMethodError: No static method itemsIndexed(Landroidx/compose/foundation/lazy/LazyListScope;Landroidx/paging/compose/LazyPagingItems;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function5;)V in class Landroidx/paging/compose/LazyPagingItemsKt; or its super classes (declaration of 'androidx.paging.compose.LazyPagingItemsKt' appears in /data/app/~~0Zxe6wFBmBefVHkKimpSbw==/in.shabinder.soundbound-c65P7RAvXrYpTikZAK2tsg==/base.apk!classes20.dex)
                                                                                                    	at app.cash.paging.LazyPagingItemsKt.itemsIndexed(LazyPagingItems.kt:31)
```
which I think is expected since I dont have any thing from androidx.paging:paging-compose, and all reference will fail.

**Comment by veyndan** [comment]
Thanks for the report! Do you mind sharing your `build.gradle` files to help me reproduce the issue on my end?

**Comment by kodeplateform** [comment]
Hello, first thanks for this library. it's really amazing. 

I'm also facing same issue by adding :                 implementation("app.cash.paging:paging-runtime-composeui:3.2.0-alpha04-0.2.0")
on commonMain from https://github.com/JetBrains/compose-multiplatform-ios-android-template/blob/main/shared/build.gradle.kts . Just to be sure : the purpose of this new library is to be able to use LazyPagingItems and collectAsLazyPagingItems() on shared code isnt it ?

**Comment by veyndan** [comment]
> Hello, first thanks for this library. it's really amazing.

Thanks!

> I'm also facing same issue by adding : implementation("app.cash.paging:paging-runtime-composeui:3.2.0-alpha04-0.2.0")
> on commonMain from https://github.com/JetBrains/compose-multiplatform-ios-android-template/blob/main/shared/build.gradle.kts .

Cool, will check it out after the long weekend and get back to you.

> Just to be sure : the purpose of this new library is to be able to use LazyPagingItems and collectAsLazyPagingItems() on shared code isnt it ?

Yep!

**Comment by veyndan** [comment]
Just published [3.2.0-alpha04-0.2.1](https://github.com/cashapp/multiplatform-paging/releases/tag/3.2.0-alpha04-0.2.1) containing a fix for this!

**Comment by kodeplateform** [comment]
Thanks works well
